source/common/runtime/runtime_features.cc

```diff
@@ -29,9 +29,9 @@ constexpr const char* runtime_features[] = {
     "envoy.reloadable_features.buffer_filter_populate_content_length",
     "envoy.reloadable_features.outlier_detection_support_for_grpc_status",
     "envoy.reloadable_features.connection_header_sanitization",
-    "envoy.reloadable_features.strict_header_validation",
```

Comment:
Did you mean to delete this? Don't we still need this since it exists in the code? Or should we clean it up in the code also?

Reply:
"Also removing a spurious "envoy.reloadable_features.strict_header_validation" as it was enabled twice."  :-P

  Github truncated but the file was
    "envoy.reloadable_features.test_feature_true",	    
    *"envoy.reloadable_features.strict_header_validation",	  *
    "envoy.reloadable_features.buffer_filter_populate_content_length",	      
   "envoy.reloadable_features.outlier_detection_support_for_grpc_status",
   "envoy.reloadable_features.connection_header_sanitization",
  *  "envoy.reloadable_features.strict_header_validation",*

Reply:
Oops sorry I should read more. :)